apps/emqx_dashboard/src/emqx_dashboard_schema.erl

```diff
@@ -55,7 +55,8 @@ fields("dashboard") ->
             )},
         {cors, fun cors/1},
         {i18n_lang, fun i18n_lang/1},
-        {bootstrap_user, ?HOCON(binary(), #{desc => ?DESC(bootstrap_user), required => false})}
```

Comment:
deprecate this filed, but not remove.

Reply:
JianBo said this change happened before the release.

Reply:
don't understand why this diff appears.

Reply:
Because this change happened on the release-v5.0.5 branch, and until release-v5.0.7 was created from the master branch, the changes in release-v5.0.5 were not merged back into master.